
export const errorCatch = err => {
  console.log(err.code);
  
}